packages/cli/src/tools/index.ts

```diff
@@ -46,6 +46,7 @@ export async function executeToolCall(
         messages: [],
         toolCallId: tool.toolCallId,
         abortSignal,
+        cwd: "", // no cwd in CLI context
```

Comment:
https://github.com/TabbyML/pochi/blob/main/packages/cli/src/tools/list-files.ts

we should migrate context.cwd to this ced?